src/ingestion/models/ingestionManager.ts

```diff
@@ -101,32 +105,42 @@ export class IngestionManager {
         err,
         metadata: { originDirectory, fileNames },
       });
-
+      trace.getActiveSpan()?.setStatus({ code: SpanStatusCode.ERROR });
```

Comment:
use the `getActiveSpan()` once and reuse it as variable

Reply:
done